plugins/mcreator-localization/lang/texts_it_IT.properties

```diff
@@ -854,6 +854,7 @@ blockly.block.entity_get_first_passenger=%2 Ottieni il primo passeggero di %1
 blockly.block.entity_get_fire_ticks=Ottieni i tick rimanenti in fiamme di %1
 blockly.block.entity_get_fly_speed=Ottieni la velocità di volo di %1
 blockly.block.entity_get_oxygen=Ottieni ossigeno di %1
+blockly.block.entity_get_percent_frozen=Ottieni la percentuale di freddo di %1
```

Comment:
Please remove changes to Italian translations. Translations are to be make on Crowdin.

Reply:
Ok